src/pages/networks/forms/NetworkDefaultACLReadOnly.tsx

```diff
@@ -0,0 +1,42 @@
+import type { FormikProps } from "formik";
+import type { FC } from "react";
+import type { NetworkFormValues } from "./NetworkForm";
+
+const NetworkDefaultACLReadOnly: FC<{
```

Comment:
This should hide when no ACL is applied

<img width="911" height="810" alt="image" src="https://github.com/user-attachments/assets/30dd33b2-5c1a-4a66-b59b-7899e73d61f5" />